template/instance.jinja

```diff
@@ -0,0 +1,31 @@
+resources:
+- type: compute.v1.instance
+  name: {{ env["name"] }}
+  properties:
+     machineType: https://www.googleapis.com/compute/v1/projects/{{ env["project"] }}/zones/{{ properties["zone"] }}/machineTypes/{{ properties["machineType"] }}
+     zone: {{ properties["zone"] }}
+     networkInterfaces:
+       - network: $(ref.yubagyte-network.selfLink)
+         subnetwork: $(ref.{{ properties["subnetwork"] }}.selfLink)
+         networkIP: $(ref.{{ properties["in-address"] }}.address)
+         accessConfigs: [natIP: $(ref.{{ properties["ex-address"] }}.address)]
+     disks:
+      - deviceName: boot
+        type: PERSISTENT
+        boot: true
+        autoDelete: true
+        initializeParams:
+          diskName: boot
+          sourceImage: https://www.googleapis.com/compute/v1/projects/centos-cloud/global/images/family/centos-7
+          diskSizeGb: 50
```

Comment:
this errors because the disk attached to the centos image looks like is only 10Gi, we need to attach disk to the instance differently.

Reply:
I have updated the script to mount the attached disk.